src/main/index.ts

```diff
@@ -128,24 +123,11 @@ app.on("ready", async () => {
   PrometheusProviderRegistry.createInstance();
   initializers.initPrometheusProviderRegistry();
 
-  const userStore = UserStore.createInstance();
```

Comment:
Moved store create/load stuff to an initializer... this was getting too bloated.